card/lib/application_job.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationJob < ActiveJob::Base
+end
```

Comment:
I'm following the upgrade guide there with introducing that class and since CoC is one of rails' core principles and a new convention for 5.2 says always inherit from Application... and never from ...::Base I thought it's better to follow that pattern.